src/main/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/ReleasePrisonerService.kt

```diff
@@ -55,7 +55,7 @@ class ReleasePrisonerService(
         bookingStatus = "C"
         livingUnitMv = null
         assignedLivingUnit = null
-        location = toLocation
+        location = getAgencyLocation(AgencyLocation.OUT).getOrThrow()
```

Comment:
The `toLocation` might be a hospital - that workaround again - but the booking location always needs to be `OUT` regardless.